stories/examples/AccordionFlush.js

```diff
@@ -2,7 +2,7 @@ import React, { useState } from 'react';
 import { Accordion, AccordionBody, AccordionHeader, AccordionItem } from 'reactstrap';
 
 const Example = (props) => {
-  const [open, setOpen] = useState();
+  const [open, setOpen] = useState("");
```

Comment:
```suggestion
  const [open, setOpen] = useState('');
```
Usually all JS code is in single quotes. I should setup some eslint rules for the project

Reply:
There was an `eslintrc.js` [file](https://github.com/illiteratewriter/reactstrap/blob/25690dcbb1b3a5109b8aac7a2a55223fe38d18b6/.eslintrc.js) which was deleted. Maybe we can get that back. Any idea why it was removed?